packages/lib-classifier/src/plugins/tasks/models/Task.js

```diff
@@ -6,10 +6,22 @@ const Task = types.model('Task', {
   // override annotation in individual task models with specific annotation types
   annotation: types.safeReference(Annotation),
   taskKey: types.identifier,
-  required: types.maybe(types.union(types.string, types.boolean)), // text task required default = false
+  required: types.optional(types.boolean, false), // text task required default = false
```

Comment:
Does anyone remember what the comment means here?

Reply:
Mentioned in a PR description here: https://github.com/zooniverse/front-end-monorepo/pull/1768#issue-684948967

Reply:
That bug (the project builder stringifying everything) was fixed a long time ago, so the comment can be deleted.